hpaction/schema.py

```diff
@@ -106,6 +121,22 @@ class Meta:
         form_class = forms.FeeWaiverPublicAssistanceForm
 
 
+@schema_registry.register_mutation
+class tenantChildren(ManyToOneUserModelFormMutation):
+    class Meta:
+        exclude_fields = ['user']
```

Comment:
It would be nice if we could have this in `ManyToOneUserModelFormMutation`, but apparently anything other than `abstract = True` in that class' `Meta` is disallowed.

Reply:
Oh, I figured out how to fix this and did so in 3fc7937.